Astral Pathfinder/Game/UIManager.cpp

```diff
@@ -0,0 +1,44 @@
+//
+//  UIManager.cpp
+//  Astral Pathfinder
+//
+//  Created by Logan Harvell on 2/21/18.
+//  Copyright © 2018 Logan Harvell, Ian Holdeman. All rights reserved.
+//
+
+
+// MARK: Header File
+#include "UIManager.hpp"
+#include "Game.hpp"
+#include "TextureManager.hpp"
+
+// MARK: Libraries and Frameworks
+#include <string>
+
+void UIManager::init() {
+  depositsBox.initDeposits();
+  fertilityBox.initFertility();
+}
+
+void UIManager::update() {
+  
+}
+
+void UIManager::render() {
+  depositsBox.render();
+  fertilityBox.render();
+}
+
+void UIManager::setText(int deposits, int fertility) {
```

Comment:
Instead of passing it a planet's fields individually, in the long run there will be multiple variables being displayed and we should just pass in a copy of the planet

Reply:
fair enough